### ISSUE TITLE
Newer docker Deamons bind to localhost

### ISSUE THREAD

**Issue by heiko-braun** [heiko-braun/microservices#3]
This invalidates the (assumed) defaults addresses. In general the docker to host environment networking seems to be conceptually difficult for many people

**Comment by daftano** [comment]
I would run docker through docker-compose (providing the docker-compose.yml), mapping the docker port on same port number on localhost; connecting to localhost will avoid the need to guess container IP.
That would also allow to run consul as standalone if for some reason (such as poor connectivity) docker or the image cannot be downloaded.

**Comment by heiko-braun** [comment]
Yes, that sounds reasonable. But maybe doing everything without docker is better approach. It adds complexity to the workshop without much gaining from it
